megamek/src/megamek/common/Aero.java

```diff
@@ -206,6 +206,11 @@
 
     private int eccmRoll = 0;
 
+    @Override
+    public boolean canFlee() {
+    	return (super.canFlee() && getAltitude() == 10 && hasMP());
```

Comment:
As per neoancient's comment, let's make sure that the Aero has at least 2 MP remaining.

Reply:
hasMP() already requires 2 MP remaining in the case of aero.  if
walkingMP is 1 then running is 2.  Only if walking is zero can they not
make it.

Reply:
Ok, fair enough. Flee generally only works at the beginning of the movement phase and cannot be combined with other orders, so the mere presence of MP should be fine for now.

Reply:
I _am_ noodling on how to force a control check if MP = 1 since they have to overthrust, but I need to look up the consequences of a failed control roll in that circumstance.